includes/admin/class-rop-admin.php

```diff
@@ -1735,65 +1739,40 @@ public static function rop_check_reached_sharing_limit( $sharing_type = 'tw' ) {
 	/**
 	 * Get the data used for the survey.
 	 *
+	 * @param array  $data The data for survey in Formbricks format.
+	 * @param string $page_slug The slug of the page.
+	 *
 	 * @return array The survey metadata.
 	 */
-	public function get_survey_metadata() {
-		$license_data = get_option( 'tweet_old_post_pro_license_data', array() );
-		$attributes   = array();
-		$user_id      = 'rop_' . ( ! empty( $license_data->key ) ? $license_data->key : preg_replace( '/[^\w\d]*/', '', get_site_url() ) ); // Use a normalized version of the site URL as a user ID for free users.
+	public function get_survey_metadata( $data, $page_slug ) {
+		$license_status = apply_filters( 'product_rop_license_status', 'invalid' );
+		$license_plan   = apply_filters( 'product_rop_license_plan', false );
+		$license_key    = apply_filters( 'product_rop_license_key', false );
+
+		$install_days_number = intval( ( time() - get_option( 'rop_first_install_date', time() ) ) / DAY_IN_SECONDS );
+
+		$data = array(
+			'environmentId' => 'clwgcs7ia03df11mgz7gh15od',
+			'attributes'    => array(
+				'license_status'      => $license_status,
+				'free_version'        => $this->version,
+				'install_days_number' => $install_days_number,
+			),
+		);
 
-		$days_since_install = round( ( time() - get_option( 'rop_first_install_date', 0 ) ) / DAY_IN_SECONDS );
-		$install_category   = 0; // Normalized value.
-		if ( 0 === $days_since_install || 1 === $days_since_install ) {
-			$install_category = 0;
-		} elseif ( 1 < $days_since_install && 8 > $days_since_install ) {
-			$install_category = 7;
-		} elseif ( 8 <= $days_since_install && 31 > $days_since_install ) {
-			$install_category = 30;
-		} elseif ( 30 < $days_since_install && 90 > $days_since_install ) {
-			$install_category = 90;
-		} elseif ( 90 <= $days_since_install ) {
-			$install_category = 91;
+		if ( ! empty( $license_plan ) ) {
```

Comment:
do you think we can include the number of accounts connected, limited to 20 let's say + if the sharing is started or not?